Eastern-Research-Group/mywaterway
# Feature/prettier in cypress folder

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3352917

## Main Changes:
* Added prettier config file and pre-commit hook in the client/package.json to run prettier on Cypress folder.
* Ran prettier on the entire Cypress folder

## Steps To Test:
1. Modify a file in the Cypress folder by adding new lines or changing single quotation marks to double quotation marks. Save the file and see if prettier fixes the formatting.
2. Do the same as above but **don't** save the file. Commit the change to a new local branch and see if prettier fixes the formatting after you commit the changed file.
